house-snark/Cargo.toml

```diff
@@ -23,18 +23,39 @@ ark-snark = { version = "^0.3.0", default-features = false }
 
 ark-r1cs-std = { version = "^0.3.0", default-features = false }
 
-ark-ed-on-bls12-381 = { version = "^0.3.0", features = ["r1cs"] }
-ark-bls12-381 = { version = "^0.3.0" }
+ark-ed-on-bls12-381 = { version = "^0.3.0", default-features = false, features = ["r1cs"] }
+ark-bls12-381 = { version = "^0.3.0", default-features = false, features = ["curve"] }
 ark-groth16 = { version = "^0.3.0", default-features = false }
 ark-gm17 = { version = "^0.3.0", default-features = false }
 ark-marlin = { version = "^0.3.0", default-features = false }
 
 [features]
-default = ["std", "all_relations"]
-std = []
-all_relations = ["deposit", "linear", "merkle_tree", "withdraw", "xor"]
-deposit = []
+default = ["full"]
+full = ["std", "cli", "linear", "merkle_tree", "xor", "shielder"]
```

Comment:
Maybe we can group `linear`, `merkle_tree` and `xor` into one `relations` feature.

Reply:
Maybe we should rename the whole crate to something else than `house-snark`? It's starting to grow into very general source of relations.